scout/server/blueprints/managed_variants/templates/managed_variants/managed_variants.html

```diff
@@ -353,21 +354,27 @@ <h5>
       chrom_options.push([chrom, chrom])
     })
     populateSelect(chromSelect, chrom_options); // populate chromosome select with chromosome options
+    // Remember user query settings
+    selectedChrom = "{{filters_form.chromosome.data|safe}}";
+    if(selectedChrom !== ""){
+      chromSelect.value = selectedChrom;
+    }
     setCytobands(); // Reset cytobands select elements
   }
 
   function set_coords_fields(){
     //Make chromosome coordinates editable if a valid chromsome is selected
     selectedChrom = chromSelect.value;
-    if (selectedChrom !== 'undefined' && selectedChrom !== "All" ){
+    if (selectedChrom && selectedChrom !== "All" ){
```

Comment:
Looks good - guess that was from the previous select?!

Reply:
Perhaps!